includes/widgets/rating.php

```diff
@@ -30,6 +30,45 @@ public function get_keywords() {
 		return [ 'star', 'rating', 'review', 'score', 'scale' ];
 	}
 
+	/**
+	 * @return void
+	 */
+	public function add_style_tab() {
```

Comment:
Is it possible to use a 'protected' function? That is the default function type that we usually use. Do you have a specific reason for using a 'public' type?

Reply:
nice catch 
This probably should be private